lms/djangoapps/course_home_api/course_metadata/views.py

```diff
@@ -89,6 +92,15 @@ def get(self, request, *args, **kwargs):
             check_if_authenticated=True,
         )
 
+        if load_access and not courseware_mfe_is_visible(
```

Comment:
We calculate `courseware_mfe_is_visible` on line 103/115. No need to calculate this twice.

Reply:
This flag also only checks the courseware side of the equation, but if you plan on moving into this endpoint with the intention of checking both sides, you should probably also check the course home side of things. I have another comment addressing this bigger question too.

Reply:
So the changes to course-home have all been backed out, and the frontend now uses the two fields that are in course-home.